spec_tests/spec/app-staging/apparmor_spec.rb

```diff
@@ -25,6 +25,35 @@
   end
 end
 
+# declare expected app-armor capabilities for apache2
+apache2_capabilities = %w(
+  dac_override
```

Comment:
Correct, the changes still need to be backported to `develop`. Once they are, this test will fail, forcing us to update the tests. Same is true of the iptables changes in #1053.